applications/blockchainia.md

```diff
@@ -0,0 +1,123 @@
+
+### Project Details
+
+#### Technologies Used
+
+- Rust
+- Substrate
+- Node.js
+- C#
+- Unity IDE
+- Adobe Creative Suite - Pixel art for games, marketing materials
+
+### Ecosystem Fit
+
+Our target audience includes game developers and gamers. We will develop a decentralized means of distribution that allows game developers to release directly to consumers. Our DAO-driven online gaming infrastructure will drive community engagement and provide free infrastructure for use by developers to take their games online. This infrastructure will enable and encourage decentralized game distribution. 
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Ed Anderson (eca20@pitt.edu)
+- Will Chastka (wchastka@tepper.cmu.edu)
+- Matt Dennis (oopmatt@gmail.com)
+
+### Contact
+
+- **Contact Name:** Edmund Anderson
+- **Contact Email:** eca20@pitt.edu
+- **Website:** https://github.com/eca20
+
+### Legal Structure
+
+- **Registered Address:** 906 Yetta Ave. Pittsburgh, PA 15212
+- **Registered Legal Entity:** Edmund Casey Anderson
+
+### Team's experience
+
+Ed Anderson 
+Ed has three years of software engineering experience developing and renovating the full stack of an enterprise scale system. He holds a B.S. in Computer Science from the University of Pittsburgh, where his Capstone requirement included implementing a MVP token concept, PittCoin, which relied on a bounty-program to connect students to community-sourced homework solutions, the tokens earned from which could be exchanged for extra credit from their professors. He also has five years of experience as the creator and manager of the Katz Business Research Center, during which he oversaw the implementation and successful completion of over one thousand market research and consumer behavior studies and focus groups. 
+
+Matt Dennis
+Matt is an entrepreneur with over 10 years of sales and operational experience in the Computer Security and Blockchain industry. He holds a B.s. in Computer Forensics and Information Security, and as certifications in multiple levels of the web3 tech stack. 
+
+Will Chastka
+Will has a passion for gaming, ever since his first gaming console, a sega Saturn, playing virtual fighter 2 and Shinobi Legends, to Starcraft and DOTA. His passion for crypto extends decentralized finance, gaming, and many other applications. He previously held position as Community and Marketing Manager for CyrptoAquatics NFT project, and holds a Master's in Business administration from the University of Pittsburgh and a M.S. in Business Analytics from Carnegie Mellon University. 
+
+We plan to leverage all of our experience and industry connections to build a heavily engaged community to gain market share through exploiting an emerging niche in the market with improved technology, our parachain. 
+
+### Team Code Repos
+
+- https://github.com/eca20/Coursework/tree/main/PittCoin
+- https://github.com/eca20/Coursework/tree/main/CS1699-Blockchain
+- https://github.com/eca20/BlockchainiaJRPG
```

Comment:
This repo seems to be private.

Reply:
updated to public

Reply:
I still cannot access https://github.com/eca20/BlockchainiaJRPG.

Reply:
fixed, sorry about that!